### ISSUE TITLE
Rockets Providers vs Vite - React-router's redirect is not working

### ISSUE THREAD

**Issue by Dias999** [conceptadev/rockets-react#249]
It seems that Router's provider is still not working correctly in the App file.
Possibly related to Vite's dedupe problem.

React-router's `redirect` is not working at the app screen in the handleRefreshTokenError.

```
handleRefreshTokenError: () => {
  toast.error("Your session has ended. Please log in again to continue.");
  redirect("/sign-in");
},
```

**Comment by Dias999** [comment]
I'm on it

**Comment by Dias999** [comment]
closed with #254
